spring-pulsar/src/main/java/org/springframework/pulsar/annotation/PulsarListenerAnnotationBeanPostProcessor.java

```diff
@@ -360,9 +354,7 @@ private void processPulsarListenerAnnotation(MethodPulsarListenerEndpoint<?> end
 			endpoint.setAutoStartup(resolveExpressionAsBoolean(autoStartup, "autoStartup"));
 		}
```

Comment:
(had to put this comment here, but the line of interest is https://github.com/spring-projects-experimental/spring-pulsar/pull/24/files#diff-babecbd1e9a205d2937e46186f42982af55c2a50a928add1b6030f4bf06f016cR249)....
```java
Collection<PulsarListener> classLevelListeners = findListenerAnnotations(targetClass);
```
Looks like we do nothing w/ the class level listeners?

Reply:
Soby: We are not using class level listener now - we can remove for now.